commands/parse/parse.go

```diff
@@ -68,7 +77,13 @@ func parseInput(ctx context.Context, fileList []string) ([]byte, error) {
 	return bundle, nil
 }
 
+func finalize(input []byte) string {
```

Comment:
I dont really think we need this method. In #152 I just make `parseInput` return a string so we can avoid the `string()` casting after getting the value. So in this case, we could just have `parseInput` itself do the endline normalization and return a string.

Reply:
umm, then you can add:
- ```output = strings.Replace(output, "\\r", "", -1)```
to pass the tests in #152, and I can close this one.

Reply:
Works for me! This line is added in https://github.com/instrumenta/conftest/pull/152/commits/be382cf90abaa9b2985cabb077184919860ccf99

Reply:
cool! thanks